environment.yml

```diff
@@ -4,8 +4,9 @@ channels:
   - default
 
 dependencies:
-  - cmake>=3.15
   - ale>=0.8.8
+  - cmake>=3.15
   - csm
-  - nlohmann_json
   - eigen
+  - gdal
```

Comment:
is gdal being used or can we get away with just proj?

Reply:
Nope, it can go